charms/slurmctld/src/constants.py

```diff
@@ -3,6 +3,9 @@
 
 """This module provides constants for the slurmctld-operator charm."""
 
+SLURMCTLD_PORT = 6817
+EXPORTER_PORT = 9092
```

Comment:
```suggestion
PROMETHEUS_EXPORTER_PORT = 9092
```

A bit more verbose, but it self documents what the port really is for.